Week1/HTML.md

```diff
@@ -0,0 +1,260 @@
+
+The three tags that you need for your HTML document are `<html>`, `<head>`, and `<body>`.
+
+1. The `<html></html>` tag is the highest level element that encloses every HTML page.
+2. The `<head></head>` tag holds meta information such as the page’s title and charset.
+3. Finally, the `<body></body>` tag encloses **all the content that appears on the page**.
+
+## How does HTML work?
+
+HTML documents end with the .html or .htm extension. You can view it using any web browser. The browser reads the HTML file and renders the content for users to view it.
+
+### Task: Build a basic HTML page
+
+- open your favourite editor and create a file called `my-first-page.html`
+- type `html` in the page and press the `tab` key, this will automatically generate an html page for you.
+
+The result should look like the following:
+
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title></title>
+  </head>
+  <body></body>
+</html>
+```
+
+_Note: We will explore all the tags later_
+
+- In the `<body>` tag add an `<h1>`
+
+Headings are defined with the `<h1>` to `<h6>` tags.
+
+`<h1>` defines the most important heading and `<h6>` defines the least important heading.
+
+Your code should look something like this:
+
+```html
+<body>
+  <h1>The Main Heading</h1>
+</body>
+```
+
+- Right click on your html file and click on `open in browser`, this should open the browser with the result of this file. You should see the h1 heading with `The Main Heading` on the page.
+
+<details>
+  <summary>If you are using Atom</summary>
+  To be able to preview your html file in a browser, install the _open-in-browsers_ package by going to:
+  
+  Atom -> Preferences -> Install
+  </details>
+
+- Let's explore this a bit more, in the browser, open the inspector. You can do so by either pressing `f12` or right clicking on the page and then inspect or pressing `ctrl+shift+i`.
+
+  - You should see be able to see the elements you created before (raw html), in the elements tab. You can edit/delete/add elements here like you did in your editor.
+
+  - Change the elements in the browser from the inspector, delete the current `h1` tag and add a `div` tag.
+
+    Find out more about [`div`](https://www.w3schools.com/tags/tag_div.asp)
+
+    _Notice_ that these changes will not be saved in your file and only affect the browser, if you wish to save them then you will have to actually do it from the file itself.
+
+- Add some more elements to in your `div`, after each step check your browser to see the changes:
+
+  - Add the `h1` tag back to your code with the content `The Main Heading`.
+  - Add an `h2` tag that has the content `Sunheading`.  
+  
+    Why are headings [important?](https://www.w3schools.com/html/html_headings.asp)
+  - Add a `p` tag with the content `First paragraph`.
+  - Add an `img` tag with an image from your laptop of your choosing.  
+  
+    More about [`img`](https://www.w3schools.com/tags/tag_img.asp).
```

Comment:
again in a comment
 > More about [`img`](https://www.w3schools.com/tags/tag_img.asp).